src/ducks/categories/CategoriesPage.jsx

```diff
@@ -90,7 +90,7 @@ class CategoriesPage extends Component {
     const { categoryName, subcategoryName } = router.params
     trackPage(
       `analyse:${categoryName ? categoryName : 'home'}${
-        subcategoryName ? `:${subcategoryName}` : ''
+        subcategoryName ? `:details` : ''
```

Comment:
We don't use the subcategoryName anymore? Is it wanted?

Reply:
Yep this is wanted.